TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -527,7 +527,14 @@ private void RefreshCustomSymbols()
             try
             {
                 _customSymbols = Tools.APIHelpers.Helpers.LoadIntrinsic(TypeCobolConfiguration.Copies, TypeCobolConfiguration.Format, DiagnosticsErrorEvent); //Refresh Intrinsics
-                _customSymbols = Tools.APIHelpers.Helpers.LoadDependencies(TypeCobolConfiguration.Dependencies, TypeCobolConfiguration.Format, _customSymbols, TypeCobolConfiguration.InputFiles, TypeCobolConfiguration.CopyFolders, DiagnosticsErrorEvent); //Refresh Dependencies
+                _customSymbols = Tools.APIHelpers.Helpers.LoadDependencies(TypeCobolConfiguration.Dependencies, TypeCobolConfiguration.Format, _customSymbols, TypeCobolConfiguration.InputFiles, 
+                    TypeCobolConfiguration.CopyFolders, DiagnosticsErrorEvent, out List<RemarksDirective.TextNameVariation> usedCopies, out List<CopyDirective> missingCopies); //Refresh Dependencies
+
+                if(missingCopies.Count > 0)
+                {
+                    MissingCopiesEvent(TypeCobolConfiguration.Dependencies?.FirstOrDefault(), new MissingCopiesEvent() { Copies = missingCopies.Select(c => c.TextName).Distinct().ToList() });
```

Comment:
Here we check if config.Dependencies is not null but not in CLI.cs line 153.

Reply:
The purpose of a MissingCopiesEvent is to trigger a copy loading client side. My question is more why do we even need to give a filepath ? @smedilol

Reply:
The MissingCopiesParams requires a TextDocumentIdentifier that's why the filePath is required.
@mayanje Can we remove the property TextDocumentIdentifier in MissingCopiesParams ?
If so, it will the subject of another issue.